src/raftstore/store/peer.rs

```diff
@@ -684,6 +735,23 @@ impl Peer {
             }
         }
 
+        // If the leader lease has been expired, local read should not be performed.
+        if self.leader_lease_expired_time.is_none() {
+            return false;
+        }
+
+        let now = clocktime::raw_now();
+        let expired_time = self.leader_lease_expired_time.unwrap();
+        if now > expired_time {
+            debug!("{} leader lease expired time {:?} is outdated",
+                   self.tag,
+                   self.leader_lease_expired_time);
+            // Reset leader lease expired time.
+            self.leader_lease_expired_time = None;
+            // Perform a consistent read to raft quorum and try to renew the leader lease.
```

Comment:
In all the comments, please use "raft" or "Raft" consistently. In my opinion,  "raft" needs to be changed to "Raft".